tools/scripts/build/ensure-env.js

```diff
@@ -58,5 +59,9 @@ if (FREECODECAMP_NODE_ENV !== 'development') {
   log('Skipping environment variable checks in development');
 }
 
+getChallengesForLang('english')
```

Comment:
```suggestion
// We are defaulting to English because the ids for the challenges are same accross all languages.
getChallengesForLang('english')
```

Adding a quick comment for future me.

Reply:
Thanks, that'll help.  Btw, is it okay to keep targeting master with these?  It's going to be a headache when it comes to rebasing.

Reply:
Yes, that is fine. We will need to start adding the current commits on `master` to `next` with a cherry-pick and then force reset the master with next when its ready instead of trying to rebase.